services/streams/streams_queue/requests.go

```diff
@@ -0,0 +1,277 @@
+package streams_queue
+
+import (
+	"encoding/json"
+	"fmt"
+	_ "github.com/leanspace/terraform-provider-leanspace/helper"
+	"github.com/leanspace/terraform-provider-leanspace/services/streams/streams"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/leanspace/terraform-provider-leanspace/provider"
+)
+
+type apiStreamQueueCreateInfo struct {
+	Command streams.Stream `json:"command"`
+}
+
+type apiStreamQueueUpdateInfo struct {
+	Command  streams.Stream `json:"command"`
+	StreamId string         `json:"streamId"`
+}
+
+type apiStreamQueueResponse struct {
+	Status   string `json:"status"`
+	StreamId string `json:"streamId"`
+}
+
+type apiStreamQueueCreationResponse struct {
+	ID string `json:"id"`
+}
+
+type StreamQueueInformation struct {
+	Status        string
+	StreamId      string
+	StreamQueueId string
+}
+
+type searchStreamQueueResponse struct {
+	Content []contentItem `json:"content"`
+}
+
+type contentItem struct {
```

Comment:
it could have a clearer name
```suggestion
type streamQueueContentItem struct {
```